src/criteria_assessment/query_thresholds.jl

```diff
@@ -173,46 +184,60 @@ function apply_criteria_thresholds(
     res = Raster(criteria_stack.Depth; data=sparse(data), missingval=0)
     return res
 end
-function apply_criteria_thresholds(
-    criteria_stack::T,
-    lookup::DataFrame,
-    ruleset::Vector{CriteriaBounds{Function}}
-)::Raster where {T}
-    # Result store
-    data = falses(size(criteria_stack))
 
-    res_lookup = trues(nrow(lookup))
+"""
+    apply_criteria_lookup(
```

Comment:
Okay for now - but lets start marking things for deprecation to ease things later.